### ISSUE TITLE
kong-dashboard error on install

### ISSUE THREAD

**Issue by dwtam** [PGBI/kong-dashboard#25]
Running on CentOS 6.7.

running npm install -g kong-dashboard, get the below error.

Downloading javascript librairies...

/root/.npm-global/lib/node_modules/kong-dashboard/lib/kong-dashboard.js:21
            if (err) throw err;
                           ^
Error: Command failed: bower ESUDO         Cannot be run with sudo

Additional error details:
Since bower is a user command, there is no need to execute it with superuser permissions.
If you're having permission errors when using bower without sudo, please spend a few minutes learning more about how your system should work and make any necessary repairs.

http://www.joyent.com/blog/installing-node-and-npm
https://gist.github.com/isaacs/579814

You can however run a command with sudo using --allow-root option

    at ChildProcess.exithandler (child_process.js:658:15)
    at ChildProcess.emit (events.js:98:17)
    at maybeClose (child_process.js:766:16)
    at Process.ChildProcess._handle.onexit (child_process.js:833:5)
npm ERR! weird error 8
npm ERR! not ok code 0

**Comment by dwtam** [comment]
Figured out the issue.

**Comment by PGBI** [comment]
Hi, sorry for the late answer. I believe you were running `npm install -g` as root, which you should not.

If it doesn't work when running the command as a regular user, you should probably fix your npm permissions: https://docs.npmjs.com/getting-started/fixing-npm-permissions

**Comment by dwtam** [comment]
Hello,

It didn't want to install as a regular user as well.  I figured out the issue.
1) After installing npm/node, I noticed CentOS rpm versions are quite old.  So I had to make sure I update them to the latest.  This corrected the esudo issue, I had.

2) But it started giving me eaccess issues.  I temporarily chmod the .config .local and .cache to proper permissions so it could install.  Once I did that kong-dashboard installed properly.

Thanks for getting back to me.
